apprentice_teacher.rb

```diff
@@ -1,21 +1,17 @@
-class ApprenticeTeacher
-  attr_reader :age, :salary, :phase, :target_raise
-  attr_accessor :name
+require_relative 'school'
+require_relative 'teachable'
+
+
+class ApprenticeTeacher < School
+  include Teachable
+
+  attr_reader :salary, :target_raise, :rating_criteria
 
   def initialize(options={})
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
+    super(age: 0, name: "")
```

Comment:
Hmm, if the user passed in an actual value for these, you're going to ignore them and always use 0 and "".  Just pass the options through (which you can do by leaving off the argument in the super call)